app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/login/LoginScreen.kt

```diff
@@ -94,7 +94,7 @@ fun LoginScreen(
             }
 
             is LoginEvent.NavigateToTwoFactorLogin -> {
-                onNavigateToTwoFactorLogin(event.emailAddress, event.password)
+                onNavigateToTwoFactorLogin(event.emailAddress, event.base64EncodedPassword)
```

Comment:
💭 realize this is grandfathered in as not tested, but should we add a test now where in the lambda we can call something like 
`LoginScreen( onNavigateToTwoFactorLogin = { email, encodedString -> 
           lambdaCalled = true
           assertTrue(encodedString.isEncoded)
  },
)
```
probably overkill since I think we would need to write that check. but just a thought

Reply:
Yea, I didn't realize this was untested. I'll circle back around and make sure these changes are covered.

Reply:
```
this could work for the check

fun isBase64Encoded(str: String): Boolean {
    return try {
        Base64.getDecoder().decode(str)
        true
    } catch (e: IllegalArgumentException) {
        false
    }
}